firestore/integration_test_internal/src/filter_test.cc

```diff
@@ -23,6 +23,8 @@ namespace {
 
 using FilterTest = FirestoreIntegrationTest;
 
+#if false
+
 TEST_F(FilterTest, CopyConstructorReturnsEqualObject) {
```

Comment:
This is OK, but a better way to do this is to rename the test to DISABLED_CopyConstructorReturnsEqualObject, this will ensure gtest reports there being 1 disabled test.

Reply:
Please create a ticket in TaskFlow to track the work required to fix this test. Then, use `GTEST_SKIP` and specify a message that includes the buganizer ticket.

e.g. https://github.com/firebase/firebase-cpp-sdk/blob/b9a53cf29fefe8e351e8b6b53a238a8365aa8993/firestore/integration_test_internal/src/source_test.cc#L402-L404

Reply:
I have updated the PR with better way of disabling tests.

This is a temporary measure until another PR fixes the problem.